core/src/test/scala/unit/kafka/coordinator/transaction/TransactionMarkerChannelManagerTest.scala

```diff
@@ -59,13 +60,15 @@ class TransactionMarkerChannelManagerTest {
     reaperEnabled = false)
   private val time = new MockTime
 
+  private val metrics = new Metrics()
```

Comment:
Hmm... this does not sound right to me: each `Metrics` object is actually a registry of metrics, and within a client we should be using a single registry only (today we use two, with the other one in admin client, but that is not ideal) which gets passed around across different modules; plus by creating this Metrics object without the `metric reporters` nothing will be reported as no one will come to pull the values.